src/components/Personalization/turbine/executeRules.js

```diff
@@ -10,26 +10,28 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
+const DEFAULT_EVENT = { moduleType: "libraryLoaded" };
+
+const getEvents = rule => {
+  const { events = [] } = rule;
+
+  return events.length === 0 ? [DEFAULT_EVENT] : events;
```

Comment:
I'll adjust Konductor to always add `libraryLoaded`

Reply:
@Aaronius since JSON response from Konductor should be consumed by Mobile as well, I am wondering if we should force Konductor to add `libraryLoaded` or should we leave it to Alloy to decide.